INSTALLATION.md

```diff
@@ -0,0 +1,142 @@
+    bridge_waitport 0
+    bridge_fd 0
+    address 10.0.252.100/22
+EOL
+
+ifup br-lbaas
+ip link add v-br-vlan type veth peer name v-br-lbaas
+ip link add link v-br-lbaas name v-br-lbaas.${VLAN_ID} type vlan id ${VLAN_ID}
+ip link set v-br-vlan up
+ip link set v-br-lbaas up
+ip link set v-br-lbaas.${VLAN_ID} up
+brctl addif br-lbaas v-br-lbaas.${VLAN_ID}
+brctl addif br-vlan v-br-vlan
+```
+
+3. Verify with `ip a` and `brctl show`
+
+### Configure OSA for rpc-octavia
+1. Add the new bridge to `/etc/openstack_deploy/openstack_user_config.yml` along with the `cidr_networks` and `used_ips`
+
+Example:
+```yaml
+cidr_networks:
+lbaas: 172.29.248.0/22
+
+used_ips:
+- "172.29.248.1,172.29.248.50"
+- "172.29.248.100"
+
+global_overrides/provider-networks…
+
+- network:
+    container_bridge: "br-lbaas"
+    container_type: "veth"
+    container_interface: "eth14"
+    host_bind_override: "eth14"
+    ip_from_q: "lbaas"
+    type: "raw"
+    net_name: "lbaas"
+    group_binds:
+        - neutron_linuxbridge_agent
+        - octavia-worker
+        - octavia-housekeeping
+        - octavia-health-manager
+```
+
+2. Configure new network in Octavia by adding and adjusting the following values to `/etc/openstack_deploy/user_osa_variables_overrides.yml`
+
+```yaml
+# Name of the Octavia management network in Neutron
+octavia_neutron_management_network_name: lbaas-mgmt
+
+# Name of the provider net in the system
+octavia_provider_network_name: vlan
+octavia_provider_segmentation_id: 111
+
+# this is the name used in openstack_user_config.yml with '_address' added
+octavia_container_network_name: lbaas_address
+
+# Network type
+octavia_provider_network_type: vlan
+
+# Network CIDR
+octavia_management_net_subnet_cidr: 10.0.252.0/22
+```
+
+3. Setup the Octavia container configuration in `/etc/openstack_deploy/conf.d/octavia.yml`. Verify volume groups (e.g. `vmvg00`) by running `vgscan` on all infra hosts. Similarly, verify ip addresses for each infra hosts with `ip a` if needed.
+
+Example:
+```yaml
+---
+
+octavia-infra_hosts:
+  infra1:
+    ip: 10.0.236.100
+    container_vars:
+      lxc_container_vg_name: vmvg00
+  infra2:
+    ip: 10.0.236.101
+    container_vars:
+      lxc_container_vg_name: vmvg00
+  infra3:
+    ip: 10.0.236.102
+    container_vars:
+      lxc_container_vg_name: vmvg00
+```
+
+## Install Octavia
+1. Clone the rpc-octavia repo by running: `cd /opt && git clone https://github.com/rcbops/rpc-octavia.git`
+
+2. Run the Octavia install script: `cd /opt/rpc-octavia && ./scripts/deploy.sh`
+
+3. Download amphora image from `http://rpc-repo.rackspace.com//images/amphora/r14.3.0/` - this image is built nightly, tested, and of the test passes uploaded.
+
+6.4 Install the image by running in the service tenant (teh octavia user belongs per default to the servcie tenant): `glance image-create --name amphora-x64-haproxy --visibility private --disk-format qcow2 --container-format bare --tags octavia-amphora-image < <image-name>`
```

Comment:
Are we able to use the `openstack` client here instead of `glance`? Also a minor typo `teh` and feel like we should be more specific about ensuring `OS_PROJECT_ID` needs to be set to the guid from the service project.